autonomous_software_pkg/src/map_publisher_2.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python3
+
+import os
+import rospy
+import tf
+import tf2_ros
+import utm
+
+from enum import Enum
+from pykml import parser
+
+from geometry_msgs.msg import Point, TransformStamped
+from visualization_msgs.msg import Marker, MarkerArray
+
+
+class PlacemarkType(Enum):
+    Contour = 1
+    Lane = 2
+    Obstacle = 3
+
+
+class MapComponent:
+    def __init__(
+        self,
+        id: int = 0,
+        type: PlacemarkType = PlacemarkType.Contour,
+        list_coordinates: list = [],
+    ):
+        self.id = id
+        self.type = type
+        self.list_coordinates = list_coordinates
```

Comment:
Could you add type hints?

Reply:
I think they are already there, aren't they?

Reply:
We can also add them in the definition of the class attributes:
```
from typing import List, Tuple

self.id: int = id
self.type: PlacemarkType = type
self.list_coordinates: List[Tuple[int, int]] = list_coordinates
```

Reply:
Thanks for pointing that. Done.